packages/cli/internal/pkg/cli/workflow/manager.go

```diff
@@ -331,15 +331,17 @@ func (m *Manager) isContextDeployed(contextName string) bool {
 		return false
 	}
 	engineStackName := awsresources.RenderContextStackName(m.projectSpec.Name, contextName, m.userId)
-	_, err := m.Cfn.GetStackStatus(engineStackName)
+	status, err := m.Cfn.GetStackStatus(engineStackName)
 	if err != nil {
 		if errors.Is(err, cfn.StackDoesNotExistError) {
 			return false
 		}
 		m.err = err
 		return false
 	}
-	return true
+
+	_, activeStatusFlag := cfn.QueryableStacksMap[status]
```

Comment:
Can you get the same info from 

```
cfn.stackDefinitions[status].activeStack
```

Reply:
We can't because the current definition of "active" is not actually "active". I am going to be raising this on Monday with some more succinct points